content/cloud/cloud.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Cloud Computing'
```

Comment:
Let's make sure the name of the folder and file match the `Title`, here. (e.g., ~`cloud`~ -> `cloud-computing`)

Reply:
Also, did we remove the folder/file for the General: [Cloud Computing](https://www.codecademy.com/resources/docs/general/cloud-computing) entry to make way for this new topic? We should also try to see what can be repurposed into a new concept entry (an idea I had was for some sort of "Cloud Types" entry that contains  [this information](https://www.codecademy.com/resources/docs/general/cloud-computing#:~:text=Types%20of%20Cloud,security%20and%20compliance.)).

Reply:
@Dusch4593, any more we want to reuse from the General entry. As you can probably tell, I wrote the new one from scratch.